packages/frontend/src/hooks/useTimeAgo.ts

```diff
@@ -13,10 +13,11 @@ export const useTimeAgo = (
     }, [dateOrString]);
 
     const getTimeAgo = useCallback(() => {
-        return formatDistanceToNowStrict(parsed, {
+        const timeAgo = formatDistanceToNow(parsed, {
             addSuffix: true,
-            roundingMethod: 'floor',
         });
+
+        return timeAgo;
```

Comment:
since this affects lots of places on the app, let's see if someone complains about this 👁️